src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -26,18 +27,34 @@ const isPodCampaign = (cartItemCategory: string): boolean =>
   cartItemCategory.includes("tt-token");
 
 /**
- * Check if pod is chargeable
+ * Check if pod is chargeable.
+ *
+ * Redemption of the first token is chargeable for passport customer.
```

Comment:
Lets add some simple tests on this function to check that the various conditions will cause this function to return true/false!